src/test/java/com/stripe/net/StripeResponseTest.java

```diff
@@ -0,0 +1,91 @@
+package com.stripe.net;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+
+import com.stripe.BaseStripeTest;
+import com.stripe.Stripe;
+import com.stripe.net.StripeResponse;
+import com.stripe.net.StripeHeaders;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
+public class StripeResponseTest extends BaseStripeTest {
+	String chargeBody;
+
+	@Before
+	public void clientId() throws IOException {
+		Stripe.clientId = "ca_test";
```

Comment:
Your tests don't rely on `Stripe.clientId` at all, so you should remove this line, rename the method, and get rid of `tearDownMockAndClientId()` entirely.